plugins/available/percol.plguin.bash

```diff
@@ -0,0 +1,13 @@
+# Install
```

Comment:
Please add the `cite about-plugin` and `about-plugin` calls at the top of the file, similar to what is used in the other plugins, e.g. here: https://github.com/Bash-it/bash-it/blob/master/plugins/available/boot2docker.plugin.bash#L1